GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -142,14 +142,16 @@ gdjs.SpriteAnimation = function(imageManager, animData)
     this.directions.length = i; //Make sure to delete already existing directions which are not used anymore.
 };
 
+//TODO create correct typedefs for all the data types here
```

Comment:
Let's keep this for a later PR if you wish?

Otherwise we can create a SpriteObjectData, as well as SpriteAnimationData, with SpriteDirectionData, with SpriteFrameData :)

Reply:
too late